Documentation/gettingstarted/minikube.rst

```diff
@@ -56,24 +50,20 @@ Install kubectl & minikube
 Install Cilium
 ==============
 
-Install Cilium as `DaemonSet
-<https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/>`_ into
-your new Kubernetes cluster. The DaemonSet will automatically install itself as
-Kubernetes CNI plugin.
+.. include:: k8s-install-download-release.rst
 
 .. note::
 
    In case of installing Cilium with CRIO, please see :ref:`crio-instructions` instructions.
 
+Install Cilium using Helm:
+
 .. parsed-literal::
 
-    kubectl create -f \ |SCM_WEB|\/install/kubernetes/quick-install.yaml
+   helm install cilium |CHART_RELEASE| --namespace kube-system
```

Comment:
While I fully understand why we do this, I really liked the fact that the old getting started guide in Cilium does not require users to install any external tools (like helm) unless they use more advanced features/configuration.

While I'm not fundamentally opposed to requiring helm in Getting Started, maybe providing a `quick-install-with-hubble.yaml` nice as well. In local testing, it seems like we can `kubectl apply` this over an existing quick-install.

If we keep Helm as part of the getting started guide, we should probably also explain how to obtain Helm (since we already explain how to obtain kubectl and minikube there).

Reply:
filed https://github.com/cilium/cilium/issues/11902

Reply:
https://github.com/cilium/cilium/pull/11907